app/views/jobs/index.html.erb

```diff
@@ -5,19 +5,25 @@
 <table>
   <thead>
   <tr>
+    <th> Status</th>
     <th> Job Name</th>
     <th> Counts </th>
     <th colspan="3"></th>
   </tr>
   </thead>
-
   <tbody>
     <% @groups.count.each do |k,v|%>
+      <% joblist = Job.where("name= ?", k).order("created_at DESC") %>
       <tr>
+        <% if joblist.first["created_at"] + joblist.first["frequency"] >= Date.today  && joblist.collect {|x| x.status == "success"}.all? %>
```

Comment:
The code should either account for joblist.first["frequency"] possibly being nil, or ensure that it never can be.  To ensure it never can be, we need to ensure that the database migration sets a default value as well as any code that creates new records.